### DISCUSSION TITLE
About Bitdefender Internet Security

### DISCUSSION THREAD

**Discussion by Lithium-Z** [Devolutions/UniGetUI#2297] [Q&A]
Hello,
I use the paid version of Bitdefender Internet Security, and I get a warning from Bitdefender every time I install WingetUI and after I install it. When I keep the WingetUI folder as an exception, it detects .tmp files in the Appdata folder as infected. Is there any way to use WingetUI with a virus program?

**Comment by marticliment** [comment]
I am afraid you should talk to BitDefender. Please pass the "infected" files through virtustotal and check that they are indeed false positives, and let BitDefender know that their detection is blocking you from using a clean program
